test/common/ssl/test_data/san_uri_cert.cfg

```diff
@@ -5,19 +5,32 @@ req_extensions = v3_req
 [req_distinguished_name]
 countryName = US
 countryName_default = US
-stateOrProvinceName = CA
-stateOrProvinceName_default = CA
+stateOrProvinceName = California
+stateOrProvinceName_default = California
 localityName = San Francisco
 localityName_default = San Francisco
-organizationalUnitName = Lyft
-organizationalUnitName_default = Lyft
+organizationName = Lyft
+organizationName_default = Lyft
+organizationalUnitName = Lyft Engineering
+organizationalUnitName_default = Lyft Engineering
 commonName = Test Server
+commonName_default = Test Server
 commonName_max  = 64
 
 [v3_req]
 basicConstraints = CA:FALSE
 keyUsage = nonRepudiation, digitalSignature, keyEncipherment
+extendedKeyUsage = clientAuth, serverAuth
 subjectAltName = @alt_names
+subjectKeyIdentifier = hash
+
+[v3_ca]
+basicConstraints = critical, CA:FALSE
+keyUsage = nonRepudiation, digitalSignature, keyEncipherment
+extendedKeyUsage = clientAuth, serverAuth
+subjectAltName = @alt_names
+subjectKeyIdentifier = hash
+authorityKeyIdentifier = keyid:always
 
 [alt_names]
 URI.1 = istio:account1.foo.cluster.local
```

Comment:
Since you are changing the cert fields, could you change this (and corresponding tests) to:
spiffe://lyft.com/foo
Istio auth has recently changed the SAN format. We have updated other certs but not this one, so it would be great to keep them consistent :)